ts-client/src/dlmm/index.ts

```diff
@@ -4284,6 +4286,163 @@ export class DLMM {
     };
   }
 
+  /**
+ * The `seedLiquidity` function create multiple grouped instructions. The grouped instructions will be either [initialize bin array + initialize position instructions] or [deposit instruction] combination.
+ * @param
+ *    - `owner`: The public key of the positions owner.
+ *    - `tokenXSeedAmount`: Token X lamport amount to be seeded to the pool.
+ *    - `tokenYSeedAmount`: Token Y lamport amount to be seeded to the pool.
+ *    - `tokenXBpsDistribution`: Token X amount distribution in bin in bps.
+ *    - `tokenYBpsDistribution`: Token Y amount distribution in bin in bps.
+ *    - `price`: Price in UI format
+ *    - `base`: Base key
+ *    - `roundingUp`: 
+ * @returns {Promise<TransactionInstruction[]>}
```

Comment:
Can you add comment to indicate that the returned instructions need to be executed sequentially if it was separated into multiple transaction?

Reply:
okay